mqml/parameter/conductresist.py

```diff
@@ -68,14 +68,14 @@ def desoverh_fpm(self, lockin_param1: Parameter, lockin_param2: Parameter) -> fl
             warnings.warn('the denominator iz zero, returning NaN')
             return np.nan
         except TypeError as typerror:
-            print('Amplification and/or voltaga divisions are not set. Set them and try again.')
+            warnings.warn('Amplification and/or voltaga divisions are not set. Set them and try again.')
             raise typerror
```

Comment:
Rather than printing a warning and then raiseing an error you should put the message into the error.

Also that error seems to be misspelled 

```
raise TypeError("Amplification and/or voltaga divisions are not set. Set them and try again.")
```

Reply:
I was thinking about the same thing to customize the error message
I will do that
Thanks for the input :)